net/nss/Portfile

```diff
@@ -178,6 +178,11 @@ if {${universal_possible} && [variant_isset universal]} {
     }
 }
 
+platform darwin 9 {
```

Comment:
no need to restrict the fix by platform — if ppc64 never built, and only ppc64 code is being altered, the patch can be always applied without fear of altering other builds

Reply:
@kencu I think it is appropriate to restrict it to Darwin, though removing 9 won’t hurt. On Linux and BSD `ppc64` will presumably work as-is, though I cannot personally confirm that. Could you check it on Debian?

Reply:
you already restricted it with the Apple test in the patch, right? 

So, that covers it.

Reply:
> you already restricted it with the Apple test in the patch, right?
> 
> So, that covers it.

Fair enough.